startHbz01ToLobidResources.sh

```diff
@@ -40,11 +40,12 @@ fi
 
 function indexFile() {
 	git checkout $BRANCH
+	git pull
```

Comment:
Since the behaviour of `git pull` (without remote and branch) depends on local config (see http://stackoverflow.com/a/6861747/18154) I think it would be better to do an explicit `git pull origin $BRANCH`.